source.py

```diff
@@ -8,21 +8,37 @@
 # ## Topic
 # *What problem are you (or your stakeholder) trying to address?*
 # 📝 <!-- Answer Below -->
+# The problem we are addressing is total energy consumption vs energy output
```

Comment:
This is a pretty interesting topic, especially considering these times that the world is shifting towards renewable energy.